python/pyarrow/parquet/__init__.py

```diff
@@ -2031,6 +2037,36 @@ def fs(self):
             FutureWarning, stacklevel=2)
         return self._metadata.fs
 
+    @property
+    def metadata_(self):
```

Comment:
Had to use something different than `metadata` so I went with `metadata_` - not sure if this is the best approach.

Reply:
What's the reason `metadata` doesn't work in this case?

Reply:
Sorted, `metadata` has to be deprecated and not `_metadata`. Thank you Joris!